### DISCUSSION TITLE
HA quits responding with RingMQTT

### DISCUSSION THREAD

**Discussion by mumaw** [tsightler/ring-mqtt#926] [Support Q&A] [answered]
I am having a problem where after I get the add-on installed I am having no problems. I then sign into Ring and my entire system goes unresponsive. I have to reboot the device and stop the add-on from turning on automatically because otherwise HA will go unresponsive again.

I am wondering a couple of things, first which Raspberry Pis is the add-on supported on? And how much ram is required? Just trying to figure out why the problem only seems to arise after the login happens. Wondering if I am running out of ram for my system which is causing it to lock up.

Thanks!

**Comment by tsightler** [comment, marked as answer]
The 4th question in the [FAQ](https://github.com/tsightler/ring-mqtt/wiki/Frequently-Asked-Questions-(FAQ)) likely answers your question.

Regarding what RPis are supported, I don't explicitly test any RPi systems anymore, but I would expect any of them with at least 2GB of RAM to be sufficient to at least start the code, assuming there's not already a bunch of other addons or integrations using resources.  My development system is only base HA with Mosquitto and ring-mqtt, and it uses about 1.1GB of RAM (350MB for ring-mqtt), so that gives some idea.

In the past some users have reported issues with RPi3 since they only have 1GB of RAM so addon startup can be quite difficult, if there are more than a single camera or two as the code pre-spawns a worker thread for each camera which can use a good bit of memory during startup, but memory usage generally comes back down quickly if it can get through startup.  Some users have had some success adding some additional swap since, by default, HAOS doesn't allocate any swap space.

**Reply by mumaw** [reply]
Thank you sir for the quick reply. I didn't see the FAQ so I appreciate you sending that my way. I think that is the exact problem I am having. I will migrate it over to my Pi4 8GB of RAM!
